proto/viam/app/v1/app.proto

```diff
@@ -640,6 +640,9 @@ message GetRobotPartLogsRequest {
   optional string page_token = 4;
   // logs of all levels are returned when the levels field is empty
   repeated string levels = 5;
+  optional google.protobuf.Timestamp start = 6;
+  optional google.protobuf.Timestamp end = 7;
+  optional int32 limit = 8;   
```

Comment:
any rhyme or reason that you can discern why we use `int32` vs `int64` in some places?

Reply:
no honestly, i cmd+f and saw int32 so I used it. Just took a look and still can't discern any patternbut it should probably be an int64 right? unless you think there is any con to having int64 over int32